includes/class-sensei-settings-api.php

```diff
@@ -1060,6 +1061,19 @@ public function enqueue_field_styles() {
 			wp_enqueue_style( 'sensei-settings-imageselectors' );
 		}
 	}
+
+	/**
+	 * Exposes the sensei settings into the js runtime.
+	 */
+	public function expose_sensei_settings() {
+		$settings = wp_json_encode( $this->get_settings() );
+		wp_register_script( 'expose-sensei-settings', '', [], \false, \false );
+		wp_enqueue_script( 'expose-sensei-settings' );
+		wp_add_inline_script(
+			'expose-sensei-settings',
```

Comment:
I think it makes more sense to be tied to the `js/admin/course-edit.js` and the `js/settings.js`. So we just add it when needed, and it's added in order to make sure the variable will be ready to use (about that, it's nice to set the last argument - `$position` - of `wp_add_inline_script` to `before`).

Reply:
Made it part of the `js/admin/course-edit.js`. e7ea927f748b6133497b36ca323fc9e463f8b60c